src/value.h

```diff
@@ -123,9 +123,9 @@ typedef struct RSValue {
     // reference to another value
     struct RSValue *ref;
   };
-  RSValueType t : 8;
-  uint32_t refcount : 23;
+  RSValueType t : 7;
   uint8_t allocated : 1;
+  uint16_t refcount;
```

Comment:
`refcount` would have 16 bits instead of 23? Is it mandatory to support atomic operations?

Reply:
Yes. See my note in the PR description